pkg/yamltemplate/filetests/ytt-library/net/parse-cidr-err.yml

```diff
@@ -0,0 +1,11 @@
+#@ load("@ytt:net", "net")
+
+ipv4:
+  cidr: #@ net.parse_cidr("not-a-valid-cidr")
+
++++
+
+ERR: 
+- net.parse_cidr: invalid CIDR address: not-a-valid-cidr
```

Comment:
Nice! I didn't know it was possible to validate errors like this in these tests.